_posts/2017-02-16-comm-call-v13.md

```diff
@@ -0,0 +1,50 @@
+3. **Jenny Bryan** will talk about the mechanics of making reproducible examples using the [reprex](https://github.com/jennybc/reprex) package. Just as important, she'll discuss the philosophy i.e. that forcing yourself to make little, rigorous, self-contained examples makes you think more clearly about programming. (20min)
+4. **Scott Chamberlain** will show you how best to ask for help on [rOpenSci packages](http://ropensci.org/packages/) (10 min)
+5. Q & A (10 min, moderated by Scott Chamberlain)
+
+Resources
+-----------------------------------------------------------------
+- Nick Tierney's [blog post about the reprex package](http://www.njtierney.com/r/rbloggers/2017/01/11/reprex-magic/) that Jenny Bryan will talk about
+- Noam Ross's analysis of the most common types of error messages R users post about on Stack Overflow, motivated by his interest in teaching problem-solving skills in R: [Common errors in R: An Empirical Investigation](https://github.com/noamross/zero-dependency-problems/blob/master/misc/stack-overflow-common-r-errors.md)
+- rOpenSci [discussion forum](https://discuss.ropensci.org/)
+
+Soon after the call, we'll post the [video](https://vimeo.com/ropensci/videos) and we plan to capture the highlights and links to resources in a follow-up blog post.
+
+Speaker bios
+-----------------------------------------------------------------
+**James (JD) Long** works in reinsurance in New York City. He builds risk models which, according to [George Box](http://www.tandfonline.com/doi/abs/10.1080/01621459.1976.10480949), are wrong. In spite of this inconvenient truth, James has proved successful at convincing his employer that his models are useful. James writes poorly commented, inefficient code using Python, R, SQL, and expletives.
+
+James grew up in Kentucky and has multiple degrees from the University of Kentucky. After being marooned on the archipelago of Bermuda for four years, James now lives in Jersey City, NJ with his Minecraft addicted 9 year old daughter and his wife, a recovering trial lawyer.
+
+James on [GitHub](https://github.com/CerebralMastication), [Twitter](https://twitter.com/CMastication)
+
+**Jenny Bryan** is a member of rOpenSci's Leadership team, a software engineer at [RStudio](https://www.rstudio.com/) and Associate Professor (on leave) in the Statistics Department at the University of British Columbia. She's a biostatistician specialized in genomics and takes a special interest and delight in data analysis and statistical computing. Jenny is known for her outstanding teaching and her use of lego and mullets (the hair, not the fish) to illustrate #rstats.
+
+Jenny on [GitHub](https://github.com/jennybc), [Twitter](https://twitter.com/jennybryan)
+
+**Scott Chamberlain** is co-Founder and technical lead at rOpenSci. PhD in X, author of rOpenSci packages XXX, answers most questions on our discussion forum.
```

Comment:
worth giving a couple of examples of popular / your fave packages. Not all attendees will know you